cloudify_agent/resources/script/linux.sh.template

```diff
@@ -137,10 +137,10 @@ export -f install_agent
 configure_virtualenv()
```

Comment:
I had also hoped we'd be able to shorten this script quite a bit. Maybe just inline this function? No need to keep talking about a "virtualenv"